tests/autoria/test_average.py

```diff
@@ -0,0 +1,54 @@
+import requests_mock
+import json
+
+from autoria.api import RiaAverageCarPrice
+
+
+class TestAverage:
+    """Tests for autoria average car price calculation."""
+
+    def test_average_minimal(self, ria_categories, ria_marks, ria_states,
+                             ria_bodystyles, ria_models, ria_cities,
+                             ria_gearboxes, ria_options, ria_fuels,
+                             ria_driver_types, ria_colors, ria_average):
+        """Car price can be calculated with minimal set of parameters."""
+        with requests_mock.Mocker() as mock:
+            mock.get('/categories',
+                     text=json.dumps(ria_categories))
+            mock.get('/categories/1/marks',
+                     text=json.dumps(ria_marks))
+            mock.get('/states',
+                     text=json.dumps(ria_states))
+            mock.get('/categories/1/bodystyles',
+                     text=json.dumps(ria_bodystyles))
+            mock.get('/categories/1/marks/1/models',
+                     text=json.dumps(ria_models))
+            mock.get('/categories/1/gearboxes',
+                     text=json.dumps(ria_gearboxes))
+            mock.get('/categories/1/options',
+                     text=json.dumps(ria_options))
+            mock.get('/fuels',
+                     text=json.dumps(ria_fuels))
+            mock.get('/categories/1/driverTypes',
+                     text=json.dumps(ria_driver_types))
+            mock.get('/colors',
+                     text=json.dumps(ria_colors))
+            # Does it mean that state should be required?
+            # Or we shouldn't try to request cities if state is not set?
+            # 
+            # Actually, both. The state should be required and we shouldn't try
```

Comment:
Don't answer to my questions in comments, instead remove the questions and add explanatory comment like `We need to do this and that here because...` (or maybe the code can be changed / improved to remove the cause of the question).